scripts/csc_nvidia_smi.nhc

```diff
@@ -25,11 +25,30 @@ function nhc_nvsmi_gather_data() {
 # Run the nvidia-smi utility and verify that all GPUs
 # are functioning properly.
 function check_nvsmi_healthmon() {
+    if [[ "$#" -eq 1 ]]; then
```

Comment:
use a default value of 0 and drop this `if`?

Reply:
the reason for the if is to still support the old behavior with no arguments.
I've modified so that the default value of NUM_GPUS = -1